### ISSUE TITLE
Setup db for table

### ISSUE THREAD

**Issue by btotharye** [btotharye/mycrofthelper#4]
Setup table at https://btotharye.github.io/mycrofthelper/failed_intents.html to populate from the dynamodb of failed intents

**Comment by highjinx411** [comment]
If we use angular you would not need a table.  just ng-repeat for failed_intent in failed_intents.. then you use html and you can use whatever.  I think a list would be fine in this case.  A table is okay too.

**Comment by btotharye** [comment]
well I'm taking the failed intents asked to the bot and recording them in a db, then I was planning to show them on the page that's what I meant here

**Comment by highjinx411** [comment]
Yes that's fine.  There will need to be an API or endpoint on the db server to be able to access the data using angular.  Angular runs on the client so that's important to note.  After we get the data to the client via the endpoint then it will be easy to write any html template to display the intent results..
something like 
<table>
<tr ng-repeat="failedIntent in FailedIntents"> 
<td>
{{failedIntent.date}}
</td>
<td>
{{failedIntent.utterance}}
</td>
</tr>
</table>

for example.   Basically it will iterate through the list and give you an object to manipulate.   A table could be okay or even a list with multiple columns.  Either way it is easy once the data is populated.

**Comment by btotharye** [comment]
ok there is a js sdk for aws to connect I'll get that setup I had been using python before to test it from the bot to add the failed utterances to the database from the bot.

**Comment by highjinx411** [comment]
Oh wait you probably do not want to do that.  The JS code runs on the client side.  We should set up an endpoint/api on the mycroft server side to get the data.  Or use the messagebus?  That could be interesting.  Usually when I want data like that I will set up something like /mycroftserver/getFailedIntents..  I wonder how easy node.js is or maybe even python has a simple webapi class?

**Comment by btotharye** [comment]
we could just probably use flask on python side or nodejs even I guess
